src/helpers/__tests__/get-insights-anonymous-user-token-test.ts

```diff
@@ -0,0 +1,31 @@
+import getInsightsAnonymousUserToken, {
+  ANONYMOUS_TOKEN_COOKIE_KEY,
+} from '../get-insights-anonymous-user-token';
+
+const DAY = 86400000; /* 1 day in ms*/
+const DATE_TOMORROW = new Date(Date.now() + DAY).toUTCString();
+const DATE_YESTERDAY = new Date(Date.now() - DAY).toUTCString();
+
+const resetCookie = cookieKey => {
+  document.cookie = `${cookieKey}=;expires=Thu, 01-Jan-1970 00:00:01 GMT;`;
```

Comment:
can you add a test for a malformed cookie, like in search-insights.js?

Reply:
excellent point, added here: a68cb7a4